Dockerfile

```diff
@@ -18,14 +18,13 @@ WORKDIR $APP_HOME
 
 ADD Gemfile* $APP_HOME/
 RUN export BUNDLER_VERSION=$(cat Gemfile.lock | tail -1 | tr -d " ") && \
-  gem install bundler
+  gem install bundler foreman
 
 RUN bundle install
 RUN yarn install --check-files
 
 ADD . $APP_HOME
 
-EXPOSE 3000
-ENV RAILS_ENV production
+EXPOSE 5000
```

Comment:
Why the port change? Locally it seems to run on 3000 too when running the Procfile file with foreman.

Reply:
default is 5000 for foreman. https://github.com/ddollar/foreman/issues/667#issue-206601005

Reply:
Interesting, then maybe we need to change `puma.rb`, because we have `port        ENV.fetch("PORT") { 3000 }`

Reply:
foreman users $PORT environment variable. I'll set that in the Dockerfile so it works